UI/AppKit/Application/ApplicationDelegate.mm

```diff
@@ -189,6 +189,24 @@ - (nonnull TabController*)createChildTab:(Web::HTML::ActivateTab)activate_tab
     return controller;
 }
 
+- (void)createTabGroup:(Vector<URL::URL> const&)urls
```

Comment:
nit: Tab groups have a pretty specific meaning in browsers, so I'd call this something different to not confuse the two. Maybe just pluralize the singular form? `createNewTabs`